mobile-app/cypress/integration/functional/wallet/balances/balances.spec.ts

```diff
@@ -259,3 +259,146 @@ context('Wallet - Balances - No balance', () => {
     cy.getByTestID('empty_balances').should('exist')
   })
 })
+
+context('Wallet - Balances - USD Value', () => {
```

Comment:
Do we have some tests on the sum of portfolio?

Reply:
I added this with check if it's within a range rather than exact amount because of DFI token rewards from LP